### PR TITLE
Mentor profile form

### PR DESCRIPTION
**Pull request by xbdirisxk**

### CODE REVIEW

File: src/components/mentorForm/PersonalInformationForm.tsx [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
+import FormInput from '@/components/FormInput';
 
 const PersonalInformationForm = () => {
   const [firstName, setFirstName] = useState('');
   const [lastName, setLastName] = useState('');
   const [email, setEmail] = useState('');
   const [about, setAbout] = useState('');
+  const [country, setCountry] = useState('');
 
+  const countries = ['Somalia', 'Somaliland', 'Ethiopia', 'Kenya', 'Djibouti'];
```
Comment (abshirahmed) [inline src/components/mentorForm/PersonalInformationForm.tsx:12]: we probably want to extend the number of countries in the list and pull in this data from an external service.

File: src/pages/profile/index.tsx [hunk 2]
```diff
@@ -1,3 +1,8 @@
+import { getAuth } from 'firebase/auth';
+import MentorForm from './mentorForm';
+
 export default function Profile() {
-  return <h1 className="my-auto text-center text-4xl">User profile page</h1>;
+  const auth = getAuth();
+  console.log('===>', auth.currentUser);
```
Comment (abshirahmed) [inline src/pages/profile/index.tsx:6]: you may want to remove these until we implement some logic to handle the user logged in state.

**Review by abshirahmed — CHANGES_REQUESTED** [review]

File: src/components/mentorForm/PersonalInformationForm.tsx (hunk 1 above)
Comment (xbdirisxk) [reply src/components/mentorForm/PersonalInformationForm.tsx:12]: yes, I used this data as placeholder but real counties/regions must come from the backend

File: src/components/DropDown.tsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+      <select
+        id="countries"
+        placeholder="blah"
+        value={value}
+        onChange={(e) => setValue(e.target.value)}
+        className="block w-full rounded-lg border border-gray-300 bg-white p-2.5 text-sm text-gray-900 focus:border-blue-500 focus:ring-blue-500 dark:border-gray-600 dark:bg-gray-700 dark:text-white dark:placeholder-gray-400 dark:focus:border-blue-500 dark:focus:ring-blue-500"
+      >
+        <option hidden></option>
+        {options.map((data) => (
+          // eslint-disable-next-line react/jsx-key
```
Comment (abshirahmed) [inline src/components/DropDown.tsx:26]: can this be achieved without having to ignore eslint?

Comment (xbdirisxk) [reply src/components/DropDown.tsx:26]: yup we can use the index of the element as key but for now we can leave it like that until the service is ready

**Pull request closed without merging**